iroh-net-report/src/reportgen/probes.rs

```diff
@@ -261,19 +265,30 @@ impl ProbePlan {
                     })
                     .expect("adding QuicIpv6 probe to a QuicAddrIpv6 probe set");
             }
-            plan.add(stun_ipv4_probes);
-            plan.add(stun_ipv6_probes);
-            plan.add(quic_ipv4_probes);
-            plan.add(quic_ipv6_probes);
+            plan.add_probes(
+                &protocols,
+                vec![
+                    stun_ipv4_probes,
+                    stun_ipv6_probes,
+                    quic_ipv4_probes,
+                    quic_ipv6_probes,
+                ],
+            );
 
             // The HTTP and ICMP probes only start after the STUN probes have had a chance.
             let mut https_probes = ProbeSet::new(ProbeProto::Https);
             let mut icmp_probes_ipv4 = ProbeSet::new(ProbeProto::IcmpV4);
             let mut icmp_probes_ipv6 = ProbeSet::new(ProbeProto::IcmpV6);
+
+            let has_priority_probes = plan.has_priority_probes();
             for attempt in 0..3 {
-                let start = *max_stun_delay.get_or_insert_with(|| plan.max_delay())
-                    + DEFAULT_INITIAL_RETRANSMIT;
-                let delay = start + DEFAULT_INITIAL_RETRANSMIT * attempt as u32;
+                let delay = if !has_priority_probes {
+                    DEFAULT_INITIAL_RETRANSMIT * attempt as u32
+                } else {
+                    let start = *max_stun_delay.get_or_insert_with(|| plan.max_delay())
```

Comment:
`max_stun_delay` is no longer a correct variable name I guess?  `max_high_prio_delay`?

Reply:
changed!